mbed_targets/mbed_target.py

```diff
@@ -76,13 +76,13 @@ def from_offline_target_entry(cls, target_entry: dict) -> "MbedTarget":
         )
 
 
-def get_target_build_attributes(mbed_target: MbedTarget, path_to_targets_json: str) -> Any:
+def get_target_build_attributes(mbed_target_board_type: str, path_to_targets_json: str) -> Any:
```

Comment:
Not part of this pr, but do you think it's worth exposing this as public api for this package? I don't know what the long term plans are.

Reply:
I think it does need to be part of the public API of the package since I need to be able to at least get the build attributes somehow from `mbed-build` but I am happy for suggestions about how best to do that.

Reply:
I'd say expose it in `__init__.py` of the package: https://github.com/ARMmbed/mbed-targets/blob/master/mbed_targets/__init__.py#L20 . 🤔

Reply:
I am tempted to create a new file `mbed_targets_build_attributes.py` (or something similar) where I mirror what is going on in `mbed_targets.py` a bit, create a `MbedTargetBuildAttributes` object and move over the `get_target_build_attributes` function.

Reply:
Oooh! Sorry I was completely misunderstanding. Absolutely 👍

Reply:
I definitely will, but I might do it after I have made the changes described above if thats ok? Maybe this PR is a bit irrelevant...

Reply:
Yeah, sorry about this. I'll close this one and do it all in one go. Thanks for the pointer to exposing in the init file.